PENDING.md

```diff
@@ -1,4 +1,17 @@
-## PENDING
+## v0.24.1 PENDING
+
+BREAKING CHANGES
+
+FEATURES
+
+IMPROVEMENTS
+
+* [store] \#1952 Update IAVL dependency to v0.10.0
+
+BUG FIXES
```

Comment:
Can you fix the pending file and put the improvements line in the already allocated section?